docs/user_guide/02b_query_engine.md

```diff
@@ -149,42 +149,34 @@ lines: 41-56
 :::
 
 :::{note}
-Query methods used for changing the operands cannot be concatenated or assigned to variables, since their Return is None. That is, the following code snippet will set `gender_lowercase` as None, and as a result, an AttributeError will be thrown:
+Query methods used for changing the operands cannot be assigned to variables, for further querying, since their _Return_ is `None`. That is, the following code snippet will set `gender_lowercase` as None, and as a result, an `AttributeError` will be thrown:
 
-```python
-# Wrong implementation
-gender_lowercase = node.attribute("gender").lowercase()
-gender_lowercase.equal_to("m")
-
-AttributeError("'NoneType' object has no attribute 'equal_to'")
-
-# Wrong implementation
-gender = node.attribute("gender")
-gender.lowercase().trim()
-gender.equal_to("m")
-
-AttributeError("'NoneType' object has no attribute 'trim'")
-
-# Correct implementation
-gender = node.attribute("gender")
-gender.lowercase()
-gender.trim()
-gender.equal_to("m")
+```{exec_error-literalinclude} scripts/02b_query_engine.py
+---
+language: python
+setup-lines: 1-6
+lines: 61-68
+---
```

Comment:
it should be configurable which type of error is expected here. In this case it would be a `PanicException`.
To fulfill the issue requirement
> A distinction between accidental errors and intended errors needs to be made